package.json

```diff
@@ -56,7 +56,8 @@
   "scripts": {
     "start": "tsc --watch",
     "build": "tsc",
-    "test": "cross-env CI=true jest --coverage",
+    "test": "cross-env CI=true jest --coverage --testPathIgnorePatterns ./test/blackbox",
+    "test:blackbox": "cross-env CI=true jest --testTimeout=100000 ./test/blackbox",
```

Comment:
You can add timeout also as 3rd argument in `test()` function :)

```js
test('should be generated in TypeScript', async function() {}, {timeout})
```

Reply:
Don't want to add that for each test 😄 Think it is better to keep it global. It is also only the CI that complains about timing out.

Reply:
I could maybe do
```
jest.setTimeout();
```

Reply:
Changed 😄